src/en/news/blog/2023/landing-page/index.md

```diff
@@ -0,0 +1,80 @@
+
+As the Ceph Dashboard continues to add new features and capabilities, the landing page has been updated to keep pace. We are excited to present the updated landing page:
+
+![Landing page](assets/landing-page.png)
+
+## Goals
+
+The goal of this project is to optimize the user experience of the landing page. This will be achieved through a combination of preserving existing features, introducing new ones, presenting information in a clear and organized manner, and crafting a user-friendly interface. By following these guidelines, we aim to make the landing page a powerful tool for monitoring the cluster with ease.
+
+To accomplish this, the main focus of development was on:
+
+- **Maintain Existing Features**
+- **Enhance Additional Features**
+- **Maintain Structured Content**
+- **User-Centered Design**
+
+## Components
+
+In this section, we will take a closer look at the individual components or "cards" that make up the layout of this dashboard.
+
+### Details card
+
+![details-card](assets/details-card.png)
+The Details Card provides an overview of the cluster configuration, displaying various critical aspects of the cluster.
+
+### Status card
+
+![status-card](assets/status-card.png)
+The Status Card provides information about the overall health of the cluster and presents it with a visual icon. Additionally, alerts regarding the cluster's state are grouped into danger and warning categories for easy interpretation.
+
+The view can be expanded to allow for viewing and scrolling through alerts, organized by status, danger and warning.
+
+![status-card-open](assets/status-card-open.png)
+
+### Capacity card
+
+![capacity-card-blue](assets/capacity-card-blue.png)
+The Capacity Card displays the total and current capacity of the cluster using a double doughnut chart. The outer ring of the chart represents the nearfull and full-ratios thresholds configured for the OSDs, while the inner blue ring indicates the used capacity. This chart helps to keep track of the total used capacity of the cluster and monitor if it remains below the configured thresholds.
+
+The used capacity, displayed with the inner blue ring, will turn yellow and red after passing the warning and danger thresholds respectively:
+
+![capacity-card-yellow](assets/capacity-card-yellow.png)
+
+### Inventory card
+
+![inventory-card](assets/inventory-card.png)
+The Inventory Card serves as an inventory of the cluster and allows you to keep track of all assets within the cluster. You can directly access each item from this card by clicking on its link, which will redirect you to the corresponding page.
+
+### Cluster utilization card
+
+![cluster-utilization-card](assets/cluster-utilization-card.png)
+The Cluster Utilization Card is populated by various charts, from top to bottom:
```

Comment:
In other card you used this first sentence to describe the goal of the card, so:
```suggestion
The Cluster Utilization Card shows you the current performance of your cluster:
```